client/html/layouts/basket/product/body-default.html

```diff
@@ -0,0 +1,114 @@
+<?php
```

Comment:
so, header-default.html becomes product-header-default.html. But main-header-default.html also becomes header-default.html? Or don't I need this last file?